examples/base/params.py

```diff
@@ -82,11 +61,16 @@ class MyParameterFrame(ParameterFrame):
 class MyDecoratedFrame:
     """A ParameterFrame made with a decorator"""
 
+    R_0: Parameter[float]
     A: Parameter[float]
 
 
-mypf = MyParameterFrame.from_dict({"A": {"value": 5, "unit": ""}})
-mydecpf = MyDecoratedFrame.from_dict({"A": {"value": 5, "unit": ""}})
+mypf = MyParameterFrame.from_dict(
+    {"A": {"value": 5, "unit": ""}, "R_0": {"value": 8, "unit": "m"}}
+)
```

Comment:
A comment here noting that an empty unit is equivalent to `"dimensionless"` may be helpful.